website/content/en/docs/olm-integration/generating-a-csv.md

```diff
@@ -1,208 +1,250 @@
-Use the `--crd-dir` flag to change this path.
-    * Custom Resources: `deploy/crds/<full group>_<version>_<kind>_cr.yaml`
-    * CustomResourceDefinitions: `deploy/crds/<full group>_<resource>_crd.yaml`
-* _For Go Operators only:_ Path to the API types root directory. The CSV generator
-also parses the [CSV markers][csv-markers] from the API type definitions
-to populate certain CSV fields. By default the API types directory is `pkg/apis/`.
-Use the `--apis-dir` flag to change this path. The CSV generator expects either
-of the following layouts for the API types directory.
-    * Mulitple groups: `<API-root-dir>/<group>/<version>/`
-    * Single groups: `<API-root-dir>/<version>/`
-* Interactive command prompts: `operator-sdk generate csv` provides the `--interactive`
-flag to optionally enable an interactive command prompt before generating a CSV
-that requests input for required and optional UI metadata fields. For example:
+Comma-separated list of keywords for your operator (required):
+> app, operator
+...
+```
 
-  ```console
-  $ operator-sdk generate csv  --csv-version 0.1.0 --interactive=true
-  INFO[0000] Generating CSV manifest version 0.1.0
+_For Go Operators only:_ the command parses [CSV markers][csv-markers] from Go API type definitions, located
+in `./api` for single group projects and `./apis` for multigroup projects, to populate certain CSV fields.
+You can set an alternative path to the API types root directory with `--apis-dir`. These markers are not available
+to Ansible or Helm project types.
 
-  Display name for the operator (required):
-  > memcached
+### ClusterServiceVersion manifests
 
-  Comma-separated list of keywords for your operator (required):
-  > app, operator
-  ...
-  ```
+CSV's are manifests that define all aspects of an Operator, from what CRDs it uses to metadata describing the
+Operator's maintainers. They are typically versioned by semver, much like Operator projects themselves;
+this version is present in both their `metadata.name` and `spec.version` fields. The CSV generator called by
+`generate <bundle|packagemanifests>` requires certain input manifests to construct a CSV manifest; all inputs
+are read when either command is invoked, along with a CSV's [base](#kustomize-files), to idempotently regenerate a CSV.
 
-  By default, the command starts an interactive prompt if some base CSV is not present,
-  either in the destination directory or a package manifests directory named with version `--from-version=<version>`.
+The following resource kinds are typically included in a CSV, which are addressed by `config/manifests/bases/kustomization.yaml`:
+  - `Role`: define Operator permissions within a namespace.
+  - `ClusterRole`: define cluster-wide Operator permissions.
+  - `Deployment`: define how the Operator's operand is run in pods.
+  - `CustomResourceDefinition` (CRD): definitions of custom objects your Operator reconciles.
```

Comment:
Nit:
`CRD` is already used above in Line 73 before the acronym is defined here.
So you might want to move the `(CRD)` abbreviation there.